__tests__/app.test.js

```diff
@@ -0,0 +1,9 @@
+
+const User = require('../src/models/users.js');
+
+describe('User Model', () => {
+  it('creates a user properly', () => {
+    const testUser = new User({username: "peter", password: "geodudeRox"})
+    expect(testUser.password).toEqual("geodudeRox")
+  });
```

Comment:
Hey Peter, would love to see some more tests written in here, this is what I would like to see tested:
- Testing that a user can sign up and sign in
- Every function that you have created
- That an user can access what is designated in their roles
- Can't access what is designated in their role
- Can hit error handling middleware